<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Inspección Visual para Control de Calidad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="widget_datosUsuario" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Usuario:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_username">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>user</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="toolButton_CerrarSesion">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Cerrar sesión</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="menuPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QToolButton" name="toolButton_MaestroRefs">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Gestor del Maestro de Referencias</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="toolButton_EntrenaMIV">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Configuración de Modelos de Inspección Visual</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="toolButton_InspeccionVisual">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Inspección Visual de Producto</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuCambiar_Idioma">
    <property name="title">
     <string>A&amp;yuda</string>
    </property>
    <addaction name="action_Cambiar_idioma"/>
    <addaction name="action_Acerca_de"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionIniciar_sesi_n"/>
    <addaction name="actionCerrar_sesi_n"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menu_M_dulos">
    <property name="title">
     <string>&amp;Módulos</string>
    </property>
    <addaction name="action_1_Maestro_de_referencias"/>
    <addaction name="action_2_Entrenamiento_de_Modelo_de_Inspecci_n_Visual"/>
    <addaction name="action_3_Inspecci_n_Visual_de_Producto"/>
   </widget>
   <widget class="QMenu" name="menu_Configuraci_n">
    <property name="title">
     <string>&amp;Configuración</string>
    </property>
    <addaction name="actionSistema"/>
    <addaction name="actionInformaci_n_de_Disco"/>
    <addaction name="actionPar_metros_FIS"/>
    <addaction name="separator"/>
    <addaction name="actionLog_de_Eventos"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menu_M_dulos"/>
   <addaction name="menu_Configuraci_n"/>
   <addaction name="menuCambiar_Idioma"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIniciar_sesi_n"/>
   <addaction name="actionCerrar_sesi_n"/>
   <addaction name="action_Cambiar_idioma"/>
   <addaction name="actionSalir"/>
   <addaction name="separator"/>
   <addaction name="actionInformaci_n_de_Disco"/>
   <addaction name="actionLog_de_Eventos"/>
  </widget>
  <action name="actionIniciar_sesi_n">
   <property name="text">
    <string>&amp;Iniciar sesión</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionCerrar_sesi_n">
   <property name="text">
    <string>&amp;Cerrar sesión</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Cambiar_idioma">
   <property name="text">
    <string>&amp;Cambiar idioma</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_Acerca_de">
   <property name="text">
    <string>&amp;Acerca de...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_1_Maestro_de_referencias">
   <property name="text">
    <string>&amp;1. Maestro de referencias</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_2_Entrenamiento_de_Modelo_de_Inspecci_n_Visual">
   <property name="text">
    <string>&amp;2. Entrenamiento de Modelo de Inspección Visual</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_3_Inspecci_n_Visual_de_Producto">
   <property name="text">
    <string>&amp;3. Inspección Visual de Producto</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSistema">
   <property name="text">
    <string>Sis&amp;tema</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLog_de_Eventos">
   <property name="text">
    <string>&amp;Log de Eventos</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionPar_metros_FIS">
   <property name="text">
    <string>&amp;Parámetros FIS</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionInformaci_n_de_Disco">
   <property name="text">
    <string>I&amp;nformación de Disco</string>
   </property>
   <property name="toolTip">
    <string>Información de Disco</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
